src/renderer/components/widgets/channels/ChannelHeader.js

```diff
@@ -175,13 +174,8 @@ export const ChannelHeader = ({
                 className={classNames({
                   [classes.title]: true,
                   [classes.bold]: true
-                })}
-              >
-                {isRegisteredUsername || !isFromZbay
-                  ? `${prefix[channelType]}${
-                    isFromZbay ? channel.name : 'unknown'
```

Comment:
Check channel header for unknown user after this change, it should display unknown.